playwright.config.ts

```diff
@@ -12,25 +12,28 @@ import { BASE } from "./config/urlProvider";
  * See https://playwright.dev/docs/test-configuration.
  */
 export default defineConfig({
+  timeout: 5 * 60 * 1000,
+  
   testDir: "./tests",
   /* Run tests in files in parallel */
   fullyParallel: true,
   /* Fail the build on CI if you accidentally left test.only in the source code. */
   forbidOnly: !!process.env.CI,
-  /* Retry on CI only */
-  retries: process.env.CI ? 2 : 0,
-  /* Opt out of parallel tests on CI. */
-  workers: process.env.CI ? 1 : undefined,
+  /* Give failing tests 1 retry attempts */
+  retries: 1,
+  /* Limit the number of workers on CI, use default locally */
+  workers: process.env.CI ? 2 : undefined,
   /* Reporter to use. See https://playwright.dev/docs/test-reporters */
-  reporter: "html",
-
+  reporter: process.env.CI ? 'dot' : 'list',
   /* Shared settings for all the projects below. See https://playwright.dev/docs/api/class-testoptions. */
   use: {
     /* Base URL to use in actions like `await page.goto('/')`. */
     baseURL: BASE,
-
-    /* Collect trace when retrying the failed test. See https://playwright.dev/docs/trace-viewer */
-    trace: "on-first-retry",
+    /* Collect trace for all tests. See https://playwright.dev/docs/trace-viewer */
+    trace: "on",
+    /* Capture screenshot after each test failure. */
+    screenshot: 'only-on-failure',
+    /* Do not run browser in headless mode. */
```

Comment:
be pretty careful with this value. Remote browser will send to you an error, because it will run in headless mode by default=) 

SO, you can leave this value locally as a headless true, but cannot use it for the remote, so you needed to leave it headless = true for the remote test runs